src/components/OpacityView.js

```diff
@@ -9,7 +9,7 @@ const propTypes = {
     /**
      * Should we dim the view
      */
-    shouldDim: PropTypes.bool.isRequired,
+    shouldDim: PropTypes.bool,
```

Comment:
The component `OpacityView` as per the name is related to opacity, you are expected to pass the `shouldDim` prop. Otherwise you would just use `View`.

Are there any instance where `shouldDim` in not passed?

Reply:
Nice catch! In this case let's wrap the `shouldDim` in `Boolean()`
```js
shouldDim={Boolean(!disabled && (state.pressed || state.hovered))}
```